db/schema.rb

```diff
@@ -97,9 +97,15 @@
     t.datetime "last_sign_in_at"
     t.inet "current_sign_in_ip"
     t.inet "last_sign_in_ip"
+    t.integer "military_status"
+    t.integer "military_branch"
+    t.string "unit"
+    t.string "notes"
+    t.boolean "accept_messages"
+    t.boolean "share_profile"
+    t.boolean "accept_notices"
```

Comment:
include all the new fields you created for veterans in the veterans model comments :)